t/unit/test_models.py

```diff
@@ -99,3 +117,43 @@ def test_celery_solar_schedules_included_as_event_choices(self):
 
         for event_choice in event_choices_values:
             assert event_choice in schedules.solar._all_events
+
+    def test_duplicate_schedules(self):
+        # Duplicates cannot be tested for solar schedules because of the
+        # unique constraints in the SolarSchedule model
+        pass
```

Comment:
I'm not a big fan of adding dead code, to be honest. There are other things that we cannot test for either, right? Does this code really add value?

Reply:
See next comment

Reply:
I checked all. Which one?

Reply:
Given the constraint is here from the very beginning (ec46367) we can indeed drop the MultipleObjectsReturned block (as well as the test method of course)

Reply:
So the test would be dropped as well, excellent. :+1: